state/model.go

```diff
@@ -1242,8 +1264,10 @@ func (m *Model) destroyOps(
 
 	var ops []txn.Op
 	modelOp := txn.Op{
-		C:      modelsC,
-		Id:     modelUUID,
+		C:  modelsC,
+		Id: modelUUID,
+		// TODO (anastasiamac 2019-4-24) when forcing, we might just consider if the document exists
+		//  instead of checking Life.
 		Assert: isAliveDoc,
```

Comment:
This can fail with --force.
If --force is used, the assert needs to be that the life equals the current life (as was done in places for force destroying units)

Reply:
Although, truthfully most of these TODOs are not TODOs but a reminder that maybe we should not get out of the loop if encountering an error but keep going with other elements.